debelistic/epicMail
# #164351273 API Endpoints and DB wtih JS Data Structures

#### What does this PR do?
- Add API Endpoints Using Data Sturcture As Database

#### Description of Task to be completed?
- Add Endpoint for users to signup
- Add Endpoint for users to signin
- Add Endpoint for users to get  received mails
- Add Endpoint for users to get unread mails
- Add Endpoint for users to get sent mails
- Add Endpoint for users to get a mail
- Add Endpoint for users to get a drafts
- Add Endpoint for users to get a mail
- Add Endpoint for users to send mail
- Test Endponits using PostMan

#### How should this be manually tested?
- See API on [Heroku](https://epicmailwithds.herokuapp.com)
- See API [Documentation](https://epicmailwithds.herokuapp.com/api-docs)

#### Any background context you want to provide?
- install dependencies to use on local machine
- run 'npm start' locally to use

#### What are the relevant pivotal tracker stories?
- PT Story link - `[#164351273](https://www.pivotaltracker.com/n/projects/2314742)`
[#164351675]
[#164351485]
[#164351475]
[#164351487]
[#164351525]
[#164351619]
[#164351611]
[#164351562]
[#164351558]
[#164473738]
